src/QuickCheckVEngine/RVFI_DII.hs

```diff
@@ -37,62 +38,123 @@
 -- SUCH DAMAGE.
 --
 
-{-|
-    Module      : QuickCheckVEngine.RVFI_DII
-    Description : The RVFI-DII interface
-
-    This module re-exports the 'QuickCheckVEngine.RVFI_DII.RVFI' and
-    'QuickCheckVEngine.RVFI_DII.DII' modules, and provides functions to send and
-    receive 'DII_Packet's and 'RVFI_Packet's over a 'Socket'.
--}
-
-module QuickCheckVEngine.RVFI_DII (
-  module QuickCheckVEngine.RVFI_DII.RVFI
-, module QuickCheckVEngine.RVFI_DII.DII
-, sendDIIPacket
-, sendDIITrace
-, recvRVFIPacket
-, recvRVFITrace
-) where
-
-import QuickCheckVEngine.RVFI_DII.RVFI
-import QuickCheckVEngine.RVFI_DII.DII
+-- |
+--    Module      : QuickCheckVEngine.RVFI_DII
+--    Description : The RVFI-DII interface
+--
+--    This module re-exports the 'QuickCheckVEngine.RVFI_DII.RVFI' and
+--    'QuickCheckVEngine.RVFI_DII.DII' modules, and provides functions to send and
+--    receive 'DII_Packet's and 'RVFI_Packet's over a 'Socket'.
+module QuickCheckVEngine.RVFI_DII
+  ( module QuickCheckVEngine.RVFI_DII.RVFI,
+    module QuickCheckVEngine.RVFI_DII.DII,
+    RvfiDiiConnection(..),
+    sendDIIPacket,
+    sendDIITrace,
+    recvRVFITrace,
+    rvfiNegotiateVersion,
+  )
+where
```

Comment:
The old code formatting was better (thought the new comment block is better).

Reply:
I agree (on both points)

Reply:
I'll rebase and remove the unrelated ormulu changes. Unfortunately it's doesn't seem configurable other than by using
{- ORMOLU_DISABLE -} comments.